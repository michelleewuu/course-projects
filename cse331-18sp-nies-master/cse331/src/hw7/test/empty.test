# Test building an empty graph.

# Load empty.tsv
LoadGraph emptyGraph empty.tsv

# Print the nodes in the graph
ListNodes emptyGraph