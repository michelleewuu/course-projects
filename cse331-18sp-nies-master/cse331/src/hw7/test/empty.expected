# Test building an empty graph.

# Load empty.tsv
loaded graph emptyGraph

# Print the nodes in the graph
emptyGraph contains: